clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -444,6 +444,66 @@ TEST_F(FormatTestComments, UnderstandsBlockComments) {
                      "  int jjj; /*b*/");
 }
 
+TEST_F(FormatTestComments,
```

Comment:
What's your reason for this test? You didn't change anything comment related, right?